packages/web-app/src/modules/earn-views/components/EarningLineChart/EarningLineChart.tsx

```diff
@@ -91,15 +103,23 @@ const _EarningLineChart = ({ classes, machines, earningsPerMachine, daysShowing,
     )
   }
 
-  const machineEarningsData = Object.keys(earningsPerMachine).map((machineId) => ({
+  const individualMachineEarningsData = Object.keys(earningsPerMachine).map((machineId) => ({
     id: machineId,
     name: machineId.substring(0, 8),
     data: normalizeEarningsPerMachineData(earningsPerMachine[machineId] as EarningWindow[], daysShowing),
   }))
 
-  useEffect(() => {
-    fetchEarningsPerMachine()
-  }, [fetchEarningsPerMachine])
+  const aggregatedMachineEarningsValue = getAggregatedMachineEarningsValue(earningsPerMachine)
+
+  const aggregateMachineEarningsData = [
+    {
+      id: 'Aggregate',
+      name: 'Aggregate',
+      data: normalizeEarningsPerMachineData(aggregatedMachineEarningsValue as EarningWindow[], daysShowing),
+    },
+  ]
+
+  const machineEarningsData = isAggregateView ? aggregateMachineEarningsData : individualMachineEarningsData
```

Comment:
`...Data` is unnecessary suffix 
```suggestion
  const machineEarnings = isAggregateView ? aggregateMachineEarnings : individualMachineEarnings
```

Reply:
Removed it